### PR TITLE
[SDAO-321] Audit fixes

### PR DESCRIPTION
**Pull request by sashasashasasha151**

### CODE REVIEW

**Review by gpevnev — APPROVED** [review]

File: contracts/BlendSwap.sol [hunk 1]
```diff
@@ -8,7 +8,7 @@ contract BlendSwap {
     mapping (bytes32 => mapping (address => Swap)) public swaps;
 
     IERC20 public blend;
-    uint public constant TEN_MINUTES = 600000;
+    uint public constant TEN_MINUTES = 600;
```
Comment (kkirka) [inline contracts/BlendSwap.sol:11]: FYI, you can just write `10 minutes` https://docs.soliditylang.org/en/v0.5.3/units-and-global-variables.html#time-units

**Review by kkirka — APPROVED** [review]
LGTM

File: contracts/BlendSwap.sol [hunk 2]
```diff
@@ -78,7 +78,7 @@ contract BlendSwap {
             "Release time in the past"
         );
 
-        swaps[secretHash] = Swap({
+        swaps[secretHash][from] = Swap({
```
Comment (georgeee) [inline contracts/BlendSwap.sol:81]: Perhaps we should remove `from` field from `Swap` structure?

File: contracts/BlendSwap.sol [hunk 3]
```diff
@@ -94,36 +94,36 @@ contract BlendSwap {
...
 
         require(
-            swaps[secretHash].confirmed == false,
+            swaps[secretHash][msg.sender].confirmed == false,
             "Confirmed swap"
         );
 
         require(
-            msg.sender == swaps[secretHash].from,
+            msg.sender == swaps[secretHash][msg.sender].from,
```
Comment (georgeee) [inline contracts/BlendSwap.sol:105]: This check seems redundant

File: contracts/BlendSwap.sol (hunk 1 above)
Comment (georgeee) [reply contracts/BlendSwap.sol:11]: @sashasashasasha151 let's follow Kirill's proposal

**Review by georgeee — CHANGES_REQUESTED** [review]
Let's remove `from` field from the `Swap` structure (but not from event)

**Review by georgeee — APPROVED** [review]

**Review by kkirka — APPROVED** [review]

**Pull request merged**